<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InspectorNetworkSourceWidget</class>
 <widget class="QWidget" name="InspectorNetworkSourceWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>185</width>
    <height>112</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutTarget">
     <item>
      <spacer name="horizontalSpacerTarget">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>55</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelTarget">
       <property name="text">
        <string>Target</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxTarget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutSource">
     <item>
      <spacer name="horizontalSpacerSource">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>52</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelSource">
       <property name="text">
        <string>Source</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditSource"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutTriggerOnNext">
     <item>
      <spacer name="horizontalSpacerTriggerOnNext1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>2</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelTriggerOnNext">
       <property name="text">
        <string>Trigger on next</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxTriggerOnNext">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacerTriggerOnNext2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacerNetworkSource">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>lineEditSource</sender>
   <signal>textChanged(QString)</signal>
   <receiver>InspectorNetworkSourceWidget</receiver>
   <slot>sourceChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>184</x>
     <y>55</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBoxTarget</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>InspectorNetworkSourceWidget</receiver>
   <slot>targetChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>151</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>184</x>
     <y>14</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxTriggerOnNext</sender>
   <signal>stateChanged(int)</signal>
   <receiver>InspectorNetworkSourceWidget</receiver>
   <slot>triggerOnNextChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>102</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>188</x>
     <y>90</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>targetChanged(int)</slot>
  <slot>sourceChanged(QString)</slot>
  <slot>triggerOnNextChanged(int)</slot>
 </slots>
</ui>
